ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/workspace/WorkspaceEventsHandler.java

```diff
@@ -401,7 +402,9 @@ protected void onMessageReceived(WorkspaceStatusEvent statusEvent) {
                     break;
 
                 case RUNNING:
-                    onWorkspaceStarted(workspaceId);
+                    if (statusEvent.getPrevStatus() == WorkspaceStatus.STARTING) {
```

Comment:
`RUNNING` may be received twice, e.g. 
```
1. STOPPED -> STARTING -> RUNNING 
2. RUNNING -> SNAPSHOTTING -> RUNNING
```
`onWorkspaceStarted(workspaceId)` obviously related to the 1st case